### ISSUE TITLE
[Bug] scoop reset failed for global installed apps

### ISSUE THREAD

**Issue by ooooo84** [ScoopInstaller/Scoop#4819]
#### Current Behavior
Here are some JDKs installed globally by scoop:
![图片](https://user-images.githubusercontent.com/4973440/158713684-8687e06b-4413-47d9-964e-99264b077171.png)

when i use `scoop reset temurin11-jdk`, it saids `ERROR 'temurin11-jdk' isn't installed`

#### Expected Behavior
Successfully reset shims and environment variables

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** 5.1.19041.1320

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-03-17T08:04:45.0044426+08:00",
    "SCOOP_REPO":  "http://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master",
    "aria2-enabled":  false,
    "aria2-config":  false,
    "proxy":  "127.0.0.1:10809"
}
```

**Comment by milsav92** [comment]
It seems that the function `installed` from `lib/core.ps1` has the following definition:
```
function installed($app, $global)
```
but in `libexec/scoop-reset.ps1` it is used as:
```
    if(!(installed $app)) {
        error "'$app' isn't installed"
        return
    }
```

Temporary solution seems to be to change `libexec/scoop-reset.ps1` to use:
```
    if(!(installed $app $global)) {
        error "'$app' isn't installed"
        return
    }
```

**Comment by ooooo84** [comment]
> It seems that the function `installed` from `lib/core.ps1` has the following definition:
> 
> ```
> function installed($app, $global)
> ```
> 
> but in `libexec/scoop-reset.ps1` it is used as:
> 
> ```
>     if(!(installed $app)) {
>         error "'$app' isn't installed"
>         return
>     }
> ```
> 
> Temporary solution seems to be to change `libexec/scoop-reset.ps1` to use:
> 
> ```
>     if(!(installed $app $global)) {
>         error "'$app' isn't installed"
>         return
>     }
> ```

it works, thank you

**Comment by rashil2000** [comment]
Hey @milsav92, can you make a PR for the fix?

**Comment by niheaven** [comment]
This has been fixed in #4798

**Comment by hgkamath** [comment]
I just penned a comment in Pull https://github.com/ScoopInstaller/Scoop/pull/4798#issuecomment-1074855060 
I didn't realize it was a accepted code review
